### PR TITLE
Tickets/dm 28939

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
 """Sphinx configuration file for an LSST stack package.
 
 This configuration only affects single-package Sphinx documentation builds.
 """
 
-from documenteer.sphinxconfig.stackconf import build_package_configs
-import lsst.ts.MTDome
-
+from documenteer.conf.pipelinespkg import *  # noqa
+import lsst.ts.salobj  # noqa
```
Comment (r-owen) [inline doc/conf.py:28]: salobj -> MTDome

File: doc/conf.py [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
 
-_g = globals()
-_g.update(
-    build_package_configs(project_name="ts_MTDome", version=lsst.ts.MTDome.__version__)
-)
+project = "ts_salobj"
```
Comment (r-owen) [inline doc/conf.py:30]: salobj -> MTDome

**Review by r-owen — APPROVED** [review]
doc/conf.py needs a minor update (salobj -> MTDome -- a mistake I have made many times).

Please do add a test_validation.py at some point (not necessarily on this ticket) to check your schema.

File: doc/conf.py (hunk 1 above)
Comment (wvreeven) [reply doc/conf.py:28]: Thanks for catching that. I changed it to the correct import.

File: doc/conf.py (hunk 2 above)
Comment (wvreeven) [reply doc/conf.py:30]: Thanks for catching that. I changed it to the correct project name.

**Pull request merged**